src/main/java/io/dropwizard/actors/RabbitmqActorBundle.java

```diff
@@ -34,4 +35,8 @@ public void initialize(Bootstrap<?> bootstrap) {
     }
 
     protected abstract RMQConfig getConfig(T t);
+
+    protected abstract MetricRegistry getMetricRegistry(T t);
+
+    protected abstract ExecutorServiceProvider getExecutorServiceProvider(T t);
```

Comment:
Reason for providing this capability is to be able to inject instrumented executor services (with custom thread names if required). Will make it non abstract and provide default implementation

Reply:
Another concern is to provide capability of creating executor service without relying on ExecutorService provider to determine exact concurrency requirements. Otherwise caller would have to understand bundle configuration and provide an executor service with relevant configuration.